### PR TITLE
Bugfix webmercator X check

### PR DESCRIPTION
**Pull request by rafaqz**

### CODE REVIEW

File: src/tiles.jl [hunk 1]
```diff
@@ -122,7 +122,7 @@ function TileGrid(bbox::Extent, zoom::Int, crs::WGS84)
     # Mercantile splits the bbox in two along the antimeridian if this happens.
     # Decide if that case should be handled here or before, also considering
     # antimeridian discussion in https://github.com/rafaqz/Extents.jl/issues/4
-    @assert bbox.X[1] < bbox.X[2]
+    @assert bbox.X[1] <= bbox.X[2]
```
Comment (visr) [inline src/tiles.jl:125]:
```suggestion
    if bbox.X[1] > bbox.X[2]
        throw(ArgumentError("TileGrid Extent has xmin > xmax"))
    end
```

**Review by visr — APPROVED** [review]
Looks good. Perhaps good to change the assert into an ArgumentError.

File: src/tiles.jl (hunk 1 above)
Comment (rafaqz) [reply src/tiles.jl:125]: Ah sorry I missed this comment

Comment (visr) [reply src/tiles.jl:125]: Ha I was too late. Though in a way it is unrelated.

**Pull request merged**